releasenotes/notes/fix-windows-permissions-flare-6c218d74688596a9.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixes a bug where on windows adding permissions would result in a deadlock due to not breaking
```

Comment:
I don't think there's a need to have a release note here, as this bug has not been included in a released version of the Agent yet.

Reply:
Ah okay, I thought it always had to be done as there is a CI check for it. Should I remove it?

Reply:
You can add the `changelog/no-changelog` label to your PR to bypass this check 😄.

Reply:
Nice, will do! Thanks.